docs/pages/docs/05-distributed-analytics/compute-centrality.md

```diff
@@ -0,0 +1,119 @@
+
+Currently, Graql supports two algorithms for computing centrality: 
+- Degree 
+- K-Core
+
+## Degree
+
+The degree of an instance gives the number of other instances directly connected to the instance. 
+For an entity, such as a `person`, this means the number of connections, 
+for example instances of a `marriage` relationship.
+The `person` with the highest degree, or the greatest number of marriages, is arguably the most "interesting"
+([Glynn Wolfe](https://en.wikipedia.org/wiki/Glynn_Wolfe)).
+
+In analyics, the `degree` algorithm computes how many arrows (edges) there are attached to instances in the graph. 
+A map is returned containing degree and instances with the degree (map<key=degree, value=set of instance>). If we call:
+
+```graql
+compute centrality; using degree;
+```
+
+On the graph below we expect the degrees of the people to be:
+
+* **Barbara Herchelroth**: 3 - two marriages and a child
+* **Jacob Young**: 2 - a marriage and a child
+* **Mary Young**: 2 - two parents
+* **John Newman**: 1 - a marriage
+
+Instances with degree = 0 won't appear in the result. Therefore, the relationship concepts
+(marriage/parentship) don't have degrees.
+
+{% include note.html content="Don't forget that a relationship can also have a degree,
+if it plays a role in another relationship." %}
+
+![A simple social network.](/images/analytics_degree_full.png)
+
+## K-Core (Coreness)
+
+Coreness is a measure that can help identify tightly interlinked groups within a network.
+An instance have coreness k if the instance belongs to a 
+[k-core](https://en.wikipedia.org/wiki/Degeneracy_(graph_theory)#k-Cores) but not to any
+(k+1)-core.
+
+We can compute the coreness centrality using the following:
+
+```graql
+compute centrality; using k-core;
+```
+
+Similar to degree, a map is returned.
+
+{% include note.html content="Relationship instances will NOT have coreness." %}
+
+Additionally, if we only care about the entities that have higher coreness, we can set the minimum value of k,
+using the following command:
+
+```graql
+compute centrality; using k-core where min-k = 10;
+```
+
+So the result map will only include entities with coreness greater than or equal to 10. 
+
+{% include note.html content="k value can not be smaller than 2" %}
+
+## Centrality within a subgraph
+
+Consider that in this graph, people with more marriages are more interesting.
+We can use the subgraph functionality to restrict the graph to only see people and who they are married to.
+Once the graph has been restricted we can determine the number of marriages by computing the degree:
+
+```graql
+compute centrality in person, marriage; using degree;
+```
+
+The result will now be:
+
+* **Barbara Herchelroth**: 2
+* **Jacob Young**: 1
+* **Mary Young**: 0
+* **John Newman**: 1
+
+The subgraph command can also be used when computing k-core centrality.
+
+## Centrality of a given type
+
+Consider the subgraph example again: because relationships also have degrees, 
```

Comment:
Can we link or describe the "subgraph example" here. Often, users would not read the whole documentation and would just jump to the section of interest

Reply:
👍